docs/getting-started/configure-integration-policy.asciidoc

```diff
@@ -19,13 +19,13 @@ To configure the integration policy:
 * <<adv-policy-settings>>
 * <<save-policy>>
 
-4. Click the **Trusted applications** tab to review the list of <<trusted-apps-ov, trusted applications>> assigned to this integration policy. On this tab, you can:
-* Expand and view a trusted application — Click the arrow next to the application's name.
-* View full details for a trusted application — Click the actions button (**...**), then select **View full details**.
-* Unassign a trusted application (Platinum or Enterprise subscription) — Click the actions button (**...**), then select **Remove from policy**. This does not delete the trusted application; this just unassigns it from the current policy.
-* Assign an existing trusted application (Platinum or Enterprise subscription) — Click **Assign trusted application to policy**, then select an application from the Assign trusted applications flyout. This view lists any existing trusted applications that aren't already assigned to the current policy.
+4. Click the **Trusted applications**, **Event filters**, and **Host isolation exceptions** tabs to review the endpoint policy artifacts assigned to this integration policy (for more information, refer to <<trusted-apps-ov>>, <<event-filters>>, and <<host-isolation-exceptions>>). On each tab, you can:
```

Comment:
@benironside I think "and" is appropriate because we want the user to click on _all_ of these tabs if they need to -- I just didn't want to have three identical steps to cover all three tabs. We could change "each tab" to plural to match; I don't think it's absolutely necessary but it works this way too:
```suggestion
4. Click the **Trusted applications**, **Event filters**, and **Host isolation exceptions** tabs to review the endpoint policy artifacts assigned to this integration policy (for more information, refer to <<trusted-apps-ov>>, <<event-filters>>, and <<host-isolation-exceptions>>). On these tabs, you can:
```